Bottom: c2f476597d83bdfccbeb29282925851fd96516b7
Top:    496f97c7bd5ac739655d0c54fe1f1943b6829a1a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-24 00:43:17 +0000

find.c: Alter LookupConnection to lookup inc./excl. rat-connections in one pass


---

diff --git a/src/find.c b/src/find.c
index bae29b9..4bbf16a 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3072,8 +3072,7 @@ ListStart (int type, void *ptr1, void *ptr2, void *ptr3, int flag)
  * also the action is marked as undoable if AndDraw is true
  */
 void
-LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int flag,
-                  bool AndRats)
+LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int connected_flag, int found_flag)
 {
   void *ptr1, *ptr2, *ptr3;
   char *name;
